frontend/src/pages/LearningCourse/ActionButton/styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import { Button } from '@mui/material';
+
+import theme from 'themeSettings';
+
+export const ActionButtonWrapper = styled('div')({
+  display: 'inline-block',
+  verticalAlign: 'top',
+  width: '155px',
+  marginLeft: '40px',
+  color: 'white',
+  [theme.breakpoints.down('xl')]: {
+    marginLeft: '38px',
+  },
+  [theme.breakpoints.down('md')]: {
+    width: '120px',
+    marginLeft: '70px',
+  },
+});
+
+export const StyledButton = styled(Button)({
+  maxWidth: '155px',
```

Comment:
Please use custom button variants from theme, for example 'medium' variant already has:

```
fontWeight: '500',
 lineHeight: '22px',
 textAlign: 'center',
```
and background color from theme palette for this variant is` #D43E41`

It will help to avoid additional code